shared/fs/footer/download.js

```diff
@@ -52,6 +52,7 @@ const stylesDownload = memoize((errored: boolean) => ({
   justifyContent: 'flex-start',
   marginLeft: globalMargins.xtiny,
   width: 140,
+  minWidth: 140, // needed to avoid shrinking
```

Comment:
What shrinking? Can you use flexShrink 0?

Reply:
When building in storybook, if I resize the window too small, the download cards shrinks. But I'm realizing now the real Electron window doesn't go that small, so I'll remove this line.